pkg/oci/oci.go

```diff
@@ -198,7 +200,7 @@ func (ociClient *OciClient) ContainsTag(tag string) (bool, *reporter.KpmEvent) {
 }
 
 // Push will push the oci artifacts to oci registry from local path
-func (ociClient *OciClient) Push(localPath, tag string) *reporter.KpmEvent {
+func (ociClient *OciClient) Push(localPath, tag string, annotations map[string]string) *reporter.KpmEvent {
```

Comment:
Put this annotation in the `PushWithAnnotations` function or using some options to do this feature.

Reply:
fixed.